<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sign">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 133, 103);
</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_text">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>140</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nomal">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>일반</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="premium">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>프리미엄</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vip">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>VIP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 133, 103);
border-radioius: 0px;
</string>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
